web/src/layouts/components/Footer.tsx

```diff
@@ -1,43 +1,109 @@
 import React from 'react'
-import { Box, Text } from 'theme-ui'
+import { Box, Button, Flex, Link, Text } from 'theme-ui'
 import Container from '../../components/Container'
 import Logo from './Logo'
-import Link from '../../components/Link'
-import ALink from '../../components/ALink'
+import SocialGithubSvg from '../../assets/Social-Github.svg'
+import SocialLinkedinSvg from '../../assets/Social-Linkedin.svg'
+import SocialFacebookSvg from '../../assets/Social-Facebook.svg'
```

Comment:
SVGs should be inside in a React component. An example can be found here:

https://github.com/tabetalt/kit/blob/master/src/icons/ArrowsIcon.tsx

Reply:
As I know this SVG icons are created via js script. Maybe we need the same?

Reply:
You can copy the script, if you'd like :)

Reply:
Yeah, but at the last time that script not worked. Did you fix that?